autobahn/asyncio/wamp.py

```diff
@@ -238,9 +246,9 @@ def accept(response):
         else:
             if self.ssl and not isSecure:
                 raise RuntimeError(
-                    'ssl argument value passed to %s conflicts with the "ws:" '
-                    'prefix of the url argument. Did you mean to use "wss:"?' %
-                    self.__class__.__name__)
+                    'ssl= argument value passed to %s conflicts with the "%s" '
+                    'prefix of the url argument. Did you mean to use "%s"?' %
+                    self.__class__.__name__,self.url_type,self.url_type_secured)
             ssl = self.ssl
```

Comment:
`flake8` is sad about the spaces around the commas .. but also this could be modernized to use `{}` and `.format(..)` instead, too.